<title> ATCGU </title><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1">
<link rel ="stylesheet" href="a.css"><link rel ="shortcut icon" href="logo.ico">
<div class="w3-top w3-show-inline-block w3-white w3-card" style="z-index:2">
	<center>
		<div class="w3-hide-small">			
			<a style="width:99;color:#464646" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap"             href="Home.html"    >Home    </a>
			<a style="width:99;color:#464646" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap"             href="Project.html" >Project </a>
			<a style="width:99;color:#464646" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap w3-pale-red" href="Design.html"  >Design  </a>
			<a style="width:99;color:#464646" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap"             href="Lab.html"     >Lab     </a>
			<a style="width:99;color:#464646" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap"             href="Team.html"    >Team    </a>
			<a style="width:99;color:#464646" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap"             href="Sponsors.html">Sponsors</a>
		</div>
	</center>
	<div class="w3-hide-large w3-hide-medium">
		<a href="Home.html" class="w3-bar-item w3-button w3-large w3-pale-red">Home</a>
		<div class="w3-right w3-dropdown-click">
			<a href="#" onclick="Acc('dc1')" class="w3-bar-item w3-button"><img src="logo.ico" style="width:100%;max-width:27"></a>
			<div id="dc1" class="w3-dropdown-content w3-bar-block w3-border" style="right:0">
				<a href="#" onclick="Acc('dc2')" class="w3-bar-item w3-button w3-medium cap">Project</a>
				<div id="dc2" class="w3-hide w3-light-grey">
					<a href="Project.html#Background" class="w3-bar-item menu w3-button cap">Background</a>
					<a href="Project.html#Problem   " class="w3-bar-item menu w3-button cap">Problem   </a>
					<a href="Project.html#Goal      " class="w3-bar-item menu w3-button cap">Goal      </a>
					<a href="Project.html#Future    " class="w3-bar-item menu w3-button cap">Future    </a></div>		
				<a href="#" onclick="Acc('dc3')" class="w3-bar-item w3-button w3-medium cap">Design</a>
				<div id="dc3" class="w3-hide w3-light-grey">
					<a href="Design.html#Idea"             class="w3-bar-item menu w3-button cap">Idea</a>
					<a href="Design.html#Concept"          class="w3-bar-item menu w3-button cap">Concept</a>
					<a href="Design.html#MolecularBiology" class="w3-bar-item menu w3-button cap">Molecular Biology</a>
					<a href="Design.html#Semiconductor"    class="w3-bar-item menu w3-button cap">Semiconductor</a></div>
				<a href="Lab.html"      class="w3-bar-item menu w3-button w3-medium cap">Lab     </a>
				<a href="Team.html"     class="w3-bar-item menu w3-button w3-medium cap">Team    </a>
				<a href="Sponsors.html" class="w3-bar-item menu w3-button w3-medium cap">Sponsors</a>
			</div>
		</div>
	</div>
</div>

<div class="w3-animate-opacity page" style="margin:0 5%">
	<div class="w3-col l2 m3 w3-card-4 w3-sidebar w3-bar-block w3-hide-small w3-animate-zoom w3-center" style="top:80;left:80;height:80vh">
		<img class="w3-image w3-padding" src="logo.ico" style="width:80%">
		<h1>Design</h1>
		<a href="#Idea"             class="w3-bar-item menu w3-button w3-large cap">Idea</a>
		<a href="#Concept"          class="w3-bar-item menu w3-button w3-large cap">Concept</a>
		<a href="#MolecularBiology" class="w3-bar-item menu w3-button w3-large cap">Molecular Biology</a>
		<a href="#Semiconductor"    class="w3-bar-item menu w3-button w3-large cap">Semiconductor</a>
	</div>
	<div>
	<div class="w3-col l9 m7 s12 w3-right w3-container">
		<div class="pt" id="Idea"></div>
		<h2>Idea</h2><hr>		
		<img class="mySlides w3-mobile" src="bridge.png"     style="width:auto; height:40%">
		<img class="mySlides w3-mobile" src="123.jpg"    style="width:auto; height:40%">		
		<p>Our project <b>"bri<font color = "red">D</font>ge a<font color = "red">N</font> er<font color = "red">A</font>"</b> was inspired by Genesis which is a fresco painting by Michelangelo. One of most famous painting among Genesis, The Creation of Adam give us different idea. Since Adam's finger and God's finger are not touching, we want to build a bridge that will allow them to get in touch with each other. In addition, we hope that <b>"bri<font color = "red">D</font>ge a<font color = "red">N</font> er<font color = "red">A</font>"</b> will build a bridge between our era and the next era.</p>
<script>
myIndex = 0; c();
function c(){
	x = document.getElementsByClassName("mySlides");
	for (i = 0; i < x.length; i++) x[i].style.display = "none";
	if (++myIndex > x.length) myIndex = 1;
	x[myIndex-1].style.display = "block";
	setTimeout(c, 5000); // Change image every 5 seconds
}
</script>
		<div class="pt" id="Concept"></div>
		<h2>Concept</h2><hr>
		<center><img src="456.PNG" class="w3-mobile" style="width:47%"></center>
		<p>In the future, the most threatening enemy of human medical science may not be cancer. It is the product of gene mutation-multi-drug resistance gene. Today, the detecting chips that are designed by our team are the products withstanding multi-drug resistance gene. With the rapid changing environment and abuse of antibiotics, the speed of bacterial gene mutation surpasses human’s imagination. For example, the gene of MSSA is induced to mecA by antibiotic becoming, the new strain type, MRSA. We use this model to combine conductive electricity and DNA specificity hybridization to achieve the fast and cost-saving way of screening samples.</p>
		<p>To solve the problems, we design the DNA chips that can test the multi-drug resistance gene specifically. There are two parts in our experimental design, semiconductor design and molecular biology design.</p>
		
		<div class="pt" id="MolecularBiology"></div>
		<h2>Molecular Biology</h2><hr>
		<p>The degree of relatedness of two species can be estimated by examining how similar their base pair sequences are. The method of assessing relatedness uses DNA hybridization. The technique of DNA hybridization is based on two principles: the first principle is that the double strands of DNA are held together by hydrogen bonds between complementary base pairs; and the second principle is that the more closely related two species are, the greater will be the number of complementary base pairs in the hybrid DNA. Hybridization of DNA is accomplished by heating strands of DNA from two different species to 86° C. This breaks the hydrogen bonds between all complementary base pairs. The result is many single-stranded segments of DNA appear. The single-stranded DNA from both species is mixed together and allowed to slowly cool. Similar strands of DNA from both species will begin to chemically join together or re-anneal with complementary base pairs by reforming hydrogen bonds. The resulted hybrid DNA is then reheated and the temperature at which the DNA becomes single-stranded is noted again.</p>
		<center><img src="designb.jpg" class="w3-mobile" style="width:70%"></center>
		<p>As the part of the molecular biology experiment, we use the primer express 3.0 software to design the primer on the probe of our chip. The condition we choose is the C + G content should be between 40 to 60 percent, because it will be hard to anneal together when C + G content is lower than 40 percent (because a G-C base pair has three hydrogen bonds and A-T pair only has two) and the Tm will be too low when C + G content is higher than 60 percent. And the Tm should be between 55 – 60 ⁰C, it will cause the self-dimer when Tm is too low and will be too hard to anneal together when Tm is too high, and the discrepancy between forward and reverse cannot exceed 2 ⁰C because the condition of forward and reverse will be too different to successfully anneal. And the length of primer should be between 18 – 32 base pairs and cannot appear as the self-dimer and hairpin. A and T should not appear at the last or the head of the primer sequence and should avoid duplicate sequence ( ex AGTTTGCA、CGTATATAG ), after designing, we use the ncbi blast to confirm our primer.</p>
		<p>Furthermore, we have two kinds of chip which different from the space of two electrodes. One of the distances is 2000 nm and deduct the target DNA 600 nm, the probe of two sides must be longer than 700 nm and shorter than 1000 nm. Take off the 80 - 150 base pairs primer we designed, each strand of probe must have at least 2000 base pairs sequence. The other distance is 1000 base pairs and deduct the target 800 - 900 base pairs, the probe of two sides is approximately 50 - 60 base pairs. In this 2000 nm or 1000 base pairs sequence, the C + G content must be higher than 40 percent too. And it is higher when they are some duplicate base pairs in our sequence ( ex AAAAA、CGCGCGCGCG ). For the sequence we designed, we not only had to focus on the conductivity but also the structure of our sequence ( ex hairpin and self-dimer ). It was a tough challenge for our design.</p>
		<ul>
			<li><h3>Sequence Design</h3></li>
			<center><img src="target.PNG" class="w3-mobile" style="width:70%"></center>
			<p>We have designed four kinds of target DNA for different control. The difference between them is the size of DNA. The first is 975 base pair, it is use to check our FF is work or not just like the positive control. The second is 1277 base pair, this sequence is exceeded the distance of the two electrodes, it can ensure that when the target DNA directly touch the electrode whether be conductive or not. To confirm the reason of being conductive is the correct sequence in the DNA or just because the sequence is too long that connect to the electrode and become a access. So we find a another1277 sequence which is not a mecA sequence that can ensure the situation that we think. The third size is 175 base pair which contain only one probe sequence since we can ensure that only one specific sequence can be conductive or not. The last size is 480 base pair, it is a sequence contained in the mecA but not in the range of to probe. It can ensure that the specific of FF.</p>
			<li><h3>Probe Design</h3></li>
		<center><img src="123.PNG" class="w3-mobile" style="width:70%">
			<p>Capture probe 1: AAAATCGATGGTAAAGGTTGGCAAAAAGATAAA  </p><img src="probe1.PNG" class="w3-mobile" style="width:70%">
		<br><br><p>Capture probe 2: AACTACGGTAACATTGATCGCAACGTTCAATTT  </p><img src="probe2.PNG" class="w3-mobile" style="width:70%">
		<br><br><p>Detector probe : 5'  ACGATAATAGCAATACAATC  3'-biotin</p><img src="detectorprobe.PNG" class="w3-mobile" style="width:50%"></center>
			<li><h3>Visibilization</h3></li>
		<center><img src="134.PNG" class="w3-mobile" style="width:70%"></center>
			<p>After the target DNA is added. Target DNA will hybridize with the two capture probes already in the chip. Added the detector probe which labelled with biotin will hybridize with the target DNA. Labelled with horseradish peroxidase (HRP), anti-biotin antibody is added. The antibody binds to any detector probe already bound to the target DNA. Finally, 3,3′,5,5′-tetramethylbenzidine (TMB) substrate is added to the chip. Using a reaction that converts the substrate into a blue product which is visible. </p>
			<p>*Washing the chip to remove unbound material before added any material.</p>
		<div class="pt" id="Semiconductor"></div>
		<h2>Semiconductor</h2><hr>
		<p>To have a better specificity of short DNA segments compare to previous biosensor, 2 electrodes of gold (Au) and graphene (G) are first designed for 2 different probe DNA surface immobilization and following a high probability in target DNA selection, which is a great breakthrough of DNA chip.</p>
			<center><img src="bi-electrodes.png" class="w3-mobile" style="width:70%"></center>
		<p>The design concept is bridging of 2 probe DNAs for 1 target DNA selection. 2 different probe DNAs could be selectively immobilized by 2 different surface functional groups on 2 different electrodes, respectively. In our design, SH group could be attached on Au directly and NH2 group could be easily treated on graphene by an Aminopropyltriethoxysilane (APTES) and glutaraldehyde (GA) immersion protocol. With the well establishment of bridge, the electrical path could be conducted and following a low resistance. This behavior could be easily measured and the signals could be transferred to a user-friendly interface, which is the strength of biosensors. The designed chip could be used to detect shorter DNA segments, which is an advantage to conventional biosensors. In the meantime, much wide range of length of target DNA could be sensed by the modification of the distance or spacing between 2 electrodes. For mecA gene of MRSA, a spacing of 2 adjacent electrodes and a vertical distance between 2 electrodes is defined as 2000 and 800 nm, respectively.</p>
			<center><img src="overall.png" class="w3-mobile" style="width:70%"></center>
                <p>To have 2 different surface functional groups on the surface of electrodes, different material for 2 electrodes are necessary. 2 of most popular functional groups, such as -SH and -NH2, are chosen first. Then Au electrode is surely selected as the first electrode [ref]. A monolayer graphene grown by low-pressure chemical vapor deposition (LPCVD), which is a well-investigated 2D material, is selected as the second electrode [ref]. </p>
		<p>To design 2 different electrodes, a layout was drawn in the AutoCAD software. For a high bridging possibility, an interdigital type of electrodes arrangement is drawn with a short spacing (2000 nm) and a long distance (4000 μm). After finished the layout design, 2 masks of each layer could be fabricated as shown in Fig. X1. [請拍兩張光罩的照片] In the photolighthography step, the alignment between 2 layers are necessary but difficult to control in fully precise spacing from chip to chip. Therefore, the second design is figured out by stacking 2 electrodes. Bottom electrode is Au and graphene on PMMA could be directly transferred on Au. PMMA could be used as an insulator and its thickness, the spacing between 2 electrodes, could be used as the bridging distance of 2 probe DNAs and 1 target DNA. Au and graphene could be connected to electrical measurement out of DNA deposition area. </p>
		<center>
			<img src="graphane1.png" class="w3-mobile" style="width: auto; height: 50%;">
			<img src="graphane2.png" class="w3-mobile" style="width: auto; height: 50%;">
			<p> </p>
			<img src="graphane3.png" class="w3-mobile" style="width: auto; height: 50%;">
			<img src="graphane4.png" class="w3-mobile" style="width: auto; height: 50%;">
		</center>
		<p>At present, due to limitations in the process, we have no way to control the distance between the two electrodes very accurately. The long distance between the electrodes will affect the length of the probe and the bonding between the probe DNA and the target DNA. As the result, we make a little adjustment on our design. We changed the original flatform to a platform design. In the platform design, graphene and Au are selected on the material of the electrode, and both are conductive materials which can be well connected to modify DNA.</p>
		<center><img src="cv.png" class="w3-mobile" style="width:70%"></center>
		<p>In detecting the connection between DNA and the electrode, cyclic voltammetry is used. Cyclic voltammetry gradually change the voltage under the same current, and the redox reaction of the sample is observed, and the result of the connection is judged by the type and change of the figure value.</p>
		</div>		
	</div>
</div><div class="w3-center w3-padding w3-col l12" style="color:#dac2c2;background-color:#464646"> © Copyright 2018 @CGU </div>
<script src="jq.js"></script><script>
	$(document).on('click', 'a[href^="#"]', function(event){event.preventDefault(); $('html, body').animate({scrollTop: $($.attr(this, 'href')).offset().top}, 700);});
	Acc=(x)=>{x = document.getElementById(x); x.className = x.className.indexOf("w3-show") == -1 ? x.className + " w3-show" : x.className.replace(" w3-show", "");}
</script>
