<title> ATCGU </title><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1"><link rel ="stylesheet" href="a.css"><link rel ="shortcut icon" href="logo.ico">
<div class="w3-top w3-white w3-card">
	<div class="w3-hide-small w3-center"><style>.menu{width:99;color:#464646}</style>
		<a class="menu w3-button w3-medium w3-padding-small"             href="Home.html"    >Home    </a>
		<a class="menu w3-button w3-medium w3-padding-small"             href="Project.html" >Project </a>
		<a class="menu w3-button w3-medium w3-padding-small w3-pale-red" href="Design.html"  >Design  </a>
		<a class="menu w3-button w3-medium w3-padding-small"             href="Lab.html"     >Lab     </a>
		<a class="menu w3-button w3-medium w3-padding-small"             href="Team.html"    >Team    </a>
		<a class="menu w3-button w3-medium w3-padding-small"             href="Sponsors.html">Sponsors</a>
	</div>
	<div class="w3-hide-large w3-hide-medium">
		<a   class="w3-bar-item w3-button w3-large w3-pale-red" href="Home.html">Home</a>
		<div class="w3-right w3-dropdown-hover">
			<a   class="w3-bar-item w3-button" href="#"><img src="logo.ico" style="width:100%;max-width:27"></a>
			<div class="w3-dropdown-content w3-bar-block w3-border" style="right:0">
				<a href="Project.html"  class="w3-bar-item w3-button w3-medium">Project </a>
				<a href="Design.html"   class="w3-bar-item w3-button w3-medium">Design  </a>
				<a href="Lab.html"      class="w3-bar-item w3-button w3-medium">Lab     </a>
				<a href="Team.html"     class="w3-bar-item w3-button w3-medium">Team    </a>
				<a href="Sponsors.html" class="w3-bar-item w3-button w3-medium">Sponsors</a>
			</div>
		</div>
	</div>
</div>
<div class="w3-animate-opacity page" style="margin:0 5%">
	<div class="w3-col l2 m3 w3-card-4 w3-sidebar w3-bar-block w3-hide-small w3-animate-zoom w3-center" style="top:80;left:80;height:80%;padding:3% 0">
		<img class="w3-image w3-padding" src="logo.ico" style="width:80%"><h1>Design</h1>
		<a href="#Idea"             class="w3-bar-item w3-button w3-large">Idea</a>
		<a href="#Concept"          class="w3-bar-item w3-button w3-large">Concept</a>
		<a href="#Design" 	    class="w3-bar-item w3-button w3-large">Design</a>
	</div>
	<div class="w3-col l9 m7 s12 w3-right w3-container">
		<div class="pt" id="Idea"></div>
		<h2>Idea</h2><hr>
		<center>
			<img class="mySlides w3-mobile" src="bridge.png" style="width:auto; height:40%">
			<img class="mySlides w3-mobile" src="123.jpg"    style="width:auto; height:40%">
		</center>		
		<p>Our project <b>"bri<font color = "red">D</font>ge a<font color = "red">N</font> er<font color = "red">A</font>"</b> was inspired by Genesis which is a fresco painting by Michelangelo. One of most famous painting among Genesis, The Creation of Adam give us different idea. Since Adam's finger and God's finger are not touching, we want to build a bridge that will allow them to get in touch with each other. In addition, we hope that <b>"bri<font color = "red">D</font>ge a<font color = "red">N</font> er<font color = "red">A</font>"</b> will build a bridge between our era and the next era.</p>
<script>
myIndex = 0; c();
function     c(){
	x = document.getElementsByClassName("mySlides");
	for (i = 0; i < x.length; i++) x[i].style.display = "none";
	if (++myIndex > x.length) myIndex = 1;
	x[myIndex-1].style.display = "";
	setTimeout(c, 3000); // Change image every 5 seconds
}
</script>
		<div class="pt" id="Concept"></div>
		<h2>Concept</h2><hr>
		<center><img style="width:85%;max-width:800" src="intro.gif"></center>
		<center><img src="456.PNG" class="w3-mobile" style="width:47%"></center>
		<p>In the future, the most threatening enemy of human medical science may not be cancer. It is the product of gene mutation-multi-drug resistance gene. Today, the detecting chips that are designed by our team are the products withstanding multi-drug resistance gene. With the rapid changing environment and abuse of antibiotics, the speed of bacterial gene mutation surpasses human’s imagination. For example, the gene of MSSA is induced to mecA by antibiotic becoming, the new strain type, MRSA. We use this model to combine conductive electricity and DNA specificity hybridization to achieve the fast and cost-saving way of screening samples.</p>
		<p>To solve the problems, we design the DNA chips that can test the multi-drug resistance gene specifically. There are two parts in our experimental design, semiconductor design and molecular biology design.</p>
		
		<div class="pt" id="Design"></div>
		<h2>Design</h2><hr>
		<p>We want to lay out a bridge-like construction to fill up the gap between God's and Adam's finger by using DNA sequences, and the ones we use here will be a drug-resistant gene. With the electric conductance of DNA, the electric signal transmitted after the DNA bridge constructed successfully.</p>
		<p>The structure of the DNA bridge is made up of five materials: Route (Electron Hole), Approach Bridge (Capture Probe),Host Bridge (Target DNA), Street Light (Detector Probe),and Bridge Pier (Finger-Cross-Finger).</p>
		<center><img src="bridg.PNG" class="w3-mobile" style="width:70%"></center>
		<ul>
			<li><h3>Route (Electron Hole)</h3></li>
			<p>In DNA, the one being responsible for conduction is called an electron hole. Of solid state physics, it refers to the lack of an electron at a covalent bond, then causes the phenomenon of leaving a vacancy on the bond.</p>
			<p>Regarding the issue of DNA electrical properties, it was proposed as early as the 1960s. Since nucleotides have a planar structure similar to a benzene ring, when these nucleotides form a proper stacked pattern (p-stack), electrons may pass through the vertical plane of the Pz electronic orbital domain, moving along the electron holes on long chain, therefore the conductive of DNA built up.</p>
			<li><h3>Approach Bridge (Capture Probe)</h3></li>
			<center><img src="designb.jpg" class="w3-mobile" style="width:70%"></center>
			<p>We design a sequence of 999 base pair from the resistance gene of mecA as the distance between the two Bridge Piers. Used to connect and support the Host bridge above, both Approach Bridges (Capture Probe) are 33mer, which are located at start and end of 999 base pair respectively.</p>
			<center><img src="probe005.PNG"    class="w3-mobile" style="width:70%"></center>        
				<p>We use the NUPACK to simulate the secondary structure of Capture probe at different temperature. At 37 °C, the Capture probe have dimer because the temperature is too low so it doesn’t be specific ( left figure). The Capture probes doesn’t form dimer because the temperature reach the Tm value. It can say that the two Capture probes are specific at 60℃ and 50℃ separately (right figure).</p>
			<center><p>Capture probe 1: AAAATCGATGGTAAAGGTTGGCAAAAAGATAAA  </p> 
				<img src="probe1.PNG" class="w3-mobile" style="width:55%"> <br><br>
				<p>Capture probe 2: AACTACGGTAACATTGATCGCAACGTTCAATTT  </p>
				<img src="probe2.PNG" class="w3-mobile" style="width:55%"><br></center>
			<li><h3>Host Bridge (Target DNA)</h3></li>
			<p>To make sure the structure of our bridge is stable, we design the target DNA as the host bridge. This is the most important element of whole bridge. If the host bridge didn’t exist, there is no pathway for the electrons transmitting to other side. To finish our blueprint, we use three conditions for our materials – target DNA. All of them are based on the capture probe we designed and their only different factor is the length of sequence. They also have their negative controls. The three different conditions one by one are equal to the distance of two bridge piers(condition A), less than the distance of two bridge piers(condition B) and more than the distance of two bridge piers(condition C). And their negative controls have same condition, too.But their materials is not correct. It is not MecA sequence so they will not work absolutely.</p>
			<center><img src="think.PNG" class="w3-mobile" style="width:70%"></center>
			<p>We all know the condition A will work because it will match to the capture probe and become the access. When at condition B, because the sequence is too short that it just connect to one of the capture probe so it will not become the access. At condition C, the long sequence cross over the bridge piers so it may be work too. But we don’t know that what reason make the bridge work. It may because the sequence in the target DNA still match with the capture probe or it just be conductive because it is too long that touch the electrode and become the access. So we use the negative control to exclude the first reason. These conditions can strongly ensure that the specific of capture probe.</p>
			<p>The following are the primers we designed for different length of Target DNA.</p>
			<style>
				table, th, td {
   					 border: 3px outset pink;
					 }
					   td {
  					 text-align: center;
					 }
			</style>
				<table style="width:100%">
				 	 <tr>
   						  <th>ID</th>
    						  <th>Sequence 5' → 3'</th> 
 					 </tr>
 					 <tr>
    						  <td>Target A (F)</td>
   						  <td>AACTACGGTAACATTGATCGCAACGTTCAATTT</td>
					 </tr>
 					 <tr>
    						  <td>Target A (R)</td>
   						  <td>TTTATCTTTTTGCCAACCTTTACCATCGATTTT</td>
 					 </tr>
 					 <tr>
    						  <td>Target B (F)</td>
   						  <td>AACTACGGTAACATTGATCGCAACGTTCAATTT</td>
 					 </tr>
 					 <tr>
    						  <td>Target B (R)</td>
   						  <td>AATTCCACATTGTTTCGGTCTAAAA</td>
 					 </tr>
 					 <tr>
    						  <td>Target C (F)</td>
   						  <td>GTTGTAGTTGTCGGGTTTGGTATA  </td>
 					 </tr>
 					 <tr>
    						  <td>Target C (R)</td>
   						  <td>TGAAAGGATCTGTACTGGGTTAATCAGT </td>
 					 </tr>
			</table>
			<li><h3>Street Light (Detector Probe)</h3></li>
			<p>We use the NUPACK to simulate the secondary structure of Capture probe at different temperature. The Detector probes doesn’t form dimer while the temperature is 37℃.Moreover, we label the biotin at the 3’ as the necessary factor for “shining”.</p>
			<center>
				<p>Detector probe : 5'  ACGATAATAGCAATACAATC  3'-biotin</p><img src="detectorprobe.PNG" class="w3-mobile" style="width:35%"></center>
			<p>After the Host Bridge built onto the Approach Bridge by hybridization, we assemble Street Light (Detector Probe) onto Host Bridge with hybridization.In order to make Street Light shine, we first add the anti-biotin antibody which labelled with horseradish peroxidase (HRP), and then add 3,3',5,5'-tetramethylbenzidine (TMB) substrate. Using a reaction that converts the substrate into a blue product which is visible. Therefore, our Street Light is shining while the Host Bridge have built onto the Approach Bridge successfully.</p>
			<center><img src="visible2.PNG"    class="w3-mobile" style="width:70%"></center>
		
		<li><h3>Bridge Pier (Finger-Cross-Finger)</h3></li>
		<p>To have a better specificity of short DNA segments compare to conventional biosensors, a novel sensor structure is designed. 2 electrodes of different material including gold (Au) and graphene (G) will be fabricated separately for 2 different capture probe (probe-A and probe-B) and its surface immobilization. Therefore, the high probability could be achieved for the target DNA by the double selection from the match of 2 capture probes. It could be a great breakthrough of DNA chip.</p>
		<center><img src="bi-electrodes.png" class="w3-mobile" style="width:70%"></center>
		<p>The design concept is bridging of 2 capture probes for 1 target DNA selection. 2 different capture probes could be selectively immobilized by 2 different surface functional groups on 2 different electrodes, respectively. In our design, SH group could be attached on Au directly and NH2 group could be easily treated on graphene by an Aminopropyltriethoxysilane (APTES) and glutaraldehyde (GA) immersion protocol. With the well establishment of bridge, the electrical path could be conducted and following a low resistance. This behavior could be easily measured and the signals could be transferred to a user-friendly interface, which is the strength of biosensors. The designed chip could be used to detect shorter DNA segments, which is an advantage to conventional biosensors. In the meantime, much wide range of length of target DNA could be sensed by the modification of the distance or spacing between 2 electrodes. For mecA gene of MRSA, a spacing of 2 adjacent electrodes and a vertical distance between 2 electrodes is defined as 2000 and 800 nm, respectively.</p>
		<center><img src="overall.png" class="w3-mobile" style="width:70%"></center>
                <p>To have 2 different surface functional groups on the surface of electrodes, different material for 2 electrodes are necessary. 2 of most popular functional groups, such as -SH and -NH2, are chosen first. Then Au electrode is surely selected as the first electrode [ref]. A monolayer graphene grown by low-pressure chemical vapor deposition (LPCVD), which is a well-investigated 2D material, is selected as the second electrode [ref]. </p>
		<p>To design 2 different electrodes, a layout was drawn in the AutoCAD software. For a high bridging possibility, an interdigital type of electrodes arrangement is drawn with a short spacing (2000 nm) and a long distance (4000 μm). </p>
		<center>
			<img src="graphane1.png" class="w3-mobile" style="width: auto; height: 50%;">
			<img src="graphane2.png" class="w3-mobile" style="width: auto; height: 50%;">
			<p> </p>
			<img src="graphane3.png" class="w3-mobile" style="width: auto; height: 50%;">
			<img src="graphane4.png" class="w3-mobile" style="width: auto; height: 50%;">
		</center>
		<p>In the photolighthography step, the alignment between 2 layers are necessary but difficult to control in fully precise spacing from chip to chip. Therefore, the second design is figured out by stacking 2 electrodes. Bottom electrode is Au and graphene on PMMA could be directly transferred on Au. PMMA could be used as an insulator and its thickness, the spacing between 2 electrodes, could be used as the bridging distance of 2 probe DNAs and 1 target DNA. Au and graphene could be connected to electrical measurement out of DNA deposition area. </p>
		<p>At present, due to limitations in the process, we have no way to control the distance between the two electrodes very accurately. The long distance between the electrodes will affect the length of the probe and the bonding between the capture probe and the target DNA. As the result, we make a little adjustment on our design. We changed the original flatform to a platform design. In the platform design, graphene and Au are selected on the material of the electrode, and both are conductive materials which can be well connected to modify DNA.</p>
		<center><img src="cv.png" class="w3-mobile" style="width:70%"></center>
		<p>In detecting the connection between DNA and the electrode, cyclic voltammetry is used. Cyclic voltammetry gradually change the voltage under the same current, and the redox reaction of the sample is observed, and the result of the connection is judged by the type and change of the figure value.</p>
		</ul>
		</div>		
	</div>
</div><div class="w3-center w3-padding w3-col l12" style="color:#dac2c2;background-color:#464646"> © Copyright 2018 @CGU </div>
<script src="jq.js"></script><script>
	$(document).on('click', 'a[href^="#"]', function(event){event.preventDefault(); $('html, body').animate({scrollTop: $($.attr(this, 'href')).offset().top}, 700);});
	Acc=(x)=>{x = document.getElementById(x); x.className = x.className.indexOf("w3-show") == -1 ? x.className + " w3-show" : x.className.replace(" w3-show", "");}
</script>
