<title> ATCGU </title><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1">
<link rel ="stylesheet" href="a.css"><link rel ="shortcut icon" href="logo.ico">
<div class="w3-top w3-show-inline-block w3-white w3-card" style="z-index:2">
	<center>
		<div class="w3-hide-small">
			<a style="width:99" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap" href="Home.html"    >Home    </a>
			<a style="width:99" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap" href="Project.html" >Project </a>
			<a style="width:99" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap" href="Design.html"  >Design  </a>
			<a style="width:99" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap" href="Lab.html"     >Lab     </a>
			<a style="width:99" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap" href="Team.html"    >Team    </a>
			<a style="width:99" class="w3-bar-item menu w3-button w3-medium w3-padding-small cap" href="Sponsors.html">Sponsors</a>
		</div>
	</center>
	<div class="w3-hide-large w3-hide-medium">
		<a href="Home.html" class="w3-bar-item w3-button w3-large w3-pale-red">Home</a>
		<div class="w3-right w3-dropdown-click">
			<a href="#" onclick="Acc('dc1')" class="w3-bar-item w3-button"><img src="logo.ico" style="width:100%;max-width:27"></a>
			<div id="dc1" class="w3-dropdown-content w3-bar-block w3-border" style="right:0">
				<a href="#" onclick="Acc('dc2')" class="w3-bar-item w3-button w3-medium cap">Project</a>
				<div id="dc2" class="w3-hide w3-light-grey">
					<a href="Project.html#Background" class="w3-bar-item menu w3-button cap">Background</a>
					<a href="Project.html#Problem   " class="w3-bar-item menu w3-button cap">Problem   </a>
					<a href="Project.html#Goal      " class="w3-bar-item menu w3-button cap">Goal      </a>
					<a href="Project.html#Future    " class="w3-bar-item menu w3-button cap">Future    </a></div>		
				<a href="#" onclick="Acc('dc3')" class="w3-bar-item w3-button w3-medium cap">Design</a>
				<div id="dc3" class="w3-hide w3-light-grey">
					<a href="Design.html#Idea"             class="w3-bar-item menu w3-button cap">Idea</a>
					<a href="Design.html#Design"           class="w3-bar-item menu w3-button cap">Design</a>
					<a href="Design.html#Semiconductor"    class="w3-bar-item menu w3-button cap">Semiconductor</a>
					<a href="Design.html#MolecularBiology" class="w3-bar-item menu w3-button cap">Molecular Biology</a></div>
				<a href="Lab.html"      class="w3-bar-item menu w3-button w3-medium cap">Lab     </a>
				<a href="Team.html"     class="w3-bar-item menu w3-button w3-medium cap">Team    </a>
				<a href="Sponsors.html" class="w3-bar-item menu w3-button w3-medium cap">Sponsors</a>
			</div>
		</div>
	</div>
</div>
<div class="w3-animate-opacity" style="margin:0 10%">
	<div class="w3-col l2 m3 w3-card-4 w3-sidebar w3-bar-block w3-hide-small w3-animate-zoom w3-center" style="top:80;height:80vh">
		<img class="w3-image w3-padding" src="logo.ico">
		<h1>Lab</h1>
		<a onclick="opL('n1')" class="w3-bar-item menu w3-button w3-medium cap menu">Molecular Biology</a>
		<a onclick="opL('n2')" class="w3-bar-item menu w3-button w3-medium cap menu">Semiconductor    </a>
		<a onclick="opL('n3')" class="w3-bar-item menu w3-button w3-medium cap menu">Detection        </a>
	</div>
	<div class="w3-col l9 m7 s12 w3-right">
		<div style="height:50"></div>
		<img src="lab.png" usemap="#I02" style="width:800px;padding-top:100" class="page">
		<map name="I02">
			<area shape="circle" na="1" coords= "212,88,150" onclick="opL('n1')">
			<area shape="circle" na="2" coords= "89,335,150" onclick="opL('n2')">
			<area shape="circle" na="3" coords="687,335,150" onclick="opL('n3')">
		</map>
		<style> #mapster_wrap_0 {margin: 0 auto;}</style>
		<script src="jquery.imagemapster.js"></script><script>
		var resizeTime  = 100, resizeDelay = 100; 
		$('img').mapster({mapKey: 'na', clickNavigate: true});
		function resize(maxWidth,maxHeight) {
			var image     = $('img'),
			    imgWidth  = image.width(),
			    imgHeight = image.height(),
			    newWidth  = 0,
			    newHeight = 0;
			if (imgWidth/maxWidth>imgHeight/maxHeight)
				newWidth  = maxWidth;
			else	newHeight = maxHeight;
			image.mapster('resize',newWidth,newHeight,resizeTime);   
		}
		function onWindowResize() {
			var curWidth  = $(window).width(),
			    curHeight = $(window).height(),
			    checking=false;
			if (checking) return;
			checking = true;
			window.setTimeout(function() {
				var newWidth  = $(window).width(),
				    newHeight = $(window).height();
				if (newWidth === curWidth && newHeight === curHeight)
					resize(newWidth,newHeight);
				checking = false;
			},resizeDelay );
		}
		$(window).bind('resize',onWindowResize);
		</script>
		
		<div id="n1" class="w3-animate-opacity page" style="display:none">
			<h1><center>Molecular Biology</h1></center>
			
			<h2>introduction</h2><hr>
			<h3>USA300</h3>
			
			<p> Methicillin-resistant Staphylococcus aureus (MRSA) is a major pathogen in hospitals and the community. It causes a wide range of infections affecting the superficial skin and soft tissue, as well invasive infections including pneumonia, bloodstream infections, bone and joint infections.</p>
			<h3>Specificity(22 v.s. 999)</h3>
			<p> With a single electrode, DNA biosensor can detect a DNA fragment only 16-30 mer, limiting by the length of probe. However, based on the properties that electron can transfer through DNA, we use two electrodes which binding the same length of probe as single electrode. It can detect more than 1000 mer by analyzing the conductance. Besides, the target DNA need to hybridize with two different probe to let electron transfer through DNA.</p>
			
			<h2>experiment</h2><hr>
			<h3>preparation of target DNA</h3>
			<ul>
				<li><h4>Genomic DNA extraction</h4></li>
				<p> Before we extraction the genomic DNA we want, we need to culture the bacteria first. We transfer the bacteria culture to the 40 ml centrifuge tube and culture it overnight ( 16-18 hours ). Alter 1 ml bacteria culture to 1.5 ml microcentrifuge tube then centrifuge a 1000 rpm at 25 ℃ for 1 minute. Add 100 ul TE buffer to the tube and pipetting the pellet until there is no suspension. Add 20 ul 100 mg/ml lysozyme and 10 mg/ml lysostaphin mixture to the tube then put the tube into the 37 ℃ incubator for 60 minutes and gently shake it every 10 minutes.</p>
				<p>After taking out, add 300 ul cell lysis buffer to break the cell membrane and put in the 60 ℃ incubator and gently shake it every 3 minutes, and cool down for 5 minutes. Add 1.5 ul RNase A and put at room temperature for 5 minutes to remove the RNA. Add 100 ul protein remove buffer and put it on the ice for 10 minutes and precool the centrifuge 10 minutes too. Centrifuge at 13000 rpm for 10 minutes at 4 ℃</p>
				<p>Place the supernatant to a new 1.5 ml microcentrifuge tube and add 300 ul isopropanol and centrifuge at 13000 for 5 minutes at 4 ℃, then remove the supernatant. Add 300 ul 70 % ethanol and centrifuge for 3 minutes at 25 ℃ then drain it at vacuum environment until there is no ethanol in the tube. Add 100 ul DNA hydration buffer and put in the 60 ℃ incubator for 60 minutes and we will finish the genomic DNA extraction.</p>
				<li><h4>Real-time polymerase chain reaction</h4></li>
				<p>Before running the qPCR machine, we have to blend unknown amount of tubes which contain 1 ul DNA sample, 10 ul KAPA SYBR ® FAST qPCR Master Mix (2X), 0.4 ul forward primer, 0.4 ul reverse primer, 0.4 ul KAPA SYBR ® FAST ROX Low Reference Dye (50X), and 7.8 ul DEPC water per tube. The kinds of DNA and primers will decide the amount of how many tubes we need to run the machine. After deploying the solution, set the machine and run it about 3 hours then will show out the result.</p>
                                <center><img src="22222.jpg" style="width:60%"></center>
				<li><h4>Polymerase chain reaction</h4></li>
				<p>The PCR experiment needs four kinds of materials, DNA sample, 2x master mix, forward primer, reverse primer and water. Mixing the material in the proportion that DNA 1ul,2x master mix 10ul,primer 0.5ul and water 8ul.Total volume is 20ul. Raising the temperature to 94°C after mixing the sample. It will make DNA be denatured. Denaturation takes 30 seconds in one cycle. After denaturing, decline the temperature to 55°C. Primers will annealing to the single strand at this temperature. The step will takes 30 seconds. Finally, raising the temperature to 72°C. At this time, the DNA polymerase and dNTPs in the 2x master mix will extend the complementary DNA. The extension time is according to the distance between the forward primer and reverse primer. For example, the mecA fragment with 2000bp will takes about 2 minutes and 500bp will takes 1 minutes. After doing these steps for 30 cycles, the length of PCR product can be checked by electrophoresis. After loading the PCR product to the 1.5% agarsoe gel, we use 120V to check the result.</p>
				<img src="2.PNG">
				<li><h4>DNA extraction</h4></li>
				<p>After PCR, we have to extract the product ( gel ) to ensure the size of the DNA is correct. First, cut down the gel which contain the band we want and transfer it to a 1.5 ml microcentrifuge tube, add 500 ul buffer B and vortex until the gel dissolve then put in the 60 ℃ incubator for 10 minutes and vortex it per 3 minutes. After 10 minutes, cool down and place the fluid the PG column and centrifuge at 14000 rcf for 30 seconds then remove the supernatant. Add buffer W1 and centrifuge at 14000 rcf for 30 seconds, and remove the supernatant. Add 600 ul buffer W2 and centrifuge at 14000 rcf for 30 seconds, and remove the supernatant. Centrifuge at 14000 rcf for 2 minutes and put the PG column in a new 1.5 ml microcentrifuge tube. Add 50 ul preheat ddH2O and put at the heating plate for 5 minutes at 55 ℃. Final, centrifuge at 14000 rcf for 3 minutes and the step of extraction is done.</P>
			<p></p></ul>
			<h3>preparation of DNA-probe</h3>
		        <h4><center>strand 1: 5'  HSC6-AAAATCGATGGTAAAGGTTGGCAAAAAGATAAA-Cy3  3'</h4></center>
		        <img src="S_8626211649680.jpg"><img src="0.PNG">
		
		        <h4><center>strand 2: 5'  Cy5-AACTACGGTAACATTGATCGCAACGTTCAATTT-AminoC7  3'</h4></center>
		        <img src="S_8626211651861.jpg"><img src="1.jpg">
		</div>
		
		<div id="n2" class="w3-animate-opacity page" style="display:none">
			<h1><center>Semiconductor</h1></center>
			<h2>introduction</h2><hr>
			<p> We used gold and graphene as the material of the electrode, and fabricated the entire chip by means of a lithography process, and finally used cyclic voltammetry to test the results.</p>
			<h2>experiment</h2><hr>
			<h3>Planar Electrode</h3>
			<p> First, the glass substrate with ITO was immersed in acetone at the ultrasonic for 5 min to remove the organic particles or dust, then it was immersed in methanol at the ultrasonic for 5 min to remove acetone, then finally it was immersed in diH2O at ultrasonic for 5 min, which the glass was dried by nitrogen gun after completing the above clean process. Testing which side is coating ITO with electric meter after taking the glass and bake it for 15 min at 120°C to prevent the residual -OH from reacting with the PR. The side of ITO was applied HMDS to enhance PR adhesion before applied PR, which used spin coater in 3000 rpm 10 s at 1 step to attach the drug and 5000 rpm 20 s at 2 step to control the thickness of the drug. Note that the two drugs cannot produce any bubbles during the process of dropping the glass substrate. Sending PR-coated ITO to do soft baking for 2 min at 90°C to let PR dry a little and enhance the adhesion. After the glass was soft baking, exposing the pattern of the mask irradiating at a 365 nm ultraviolet light with a mercury lamp for 2.5 s. The way to use that is positive PR, in other words, the part is not covered by the mask and directly contacts the UV light will be eroded. Send the glass to do bake after exposure for 1 min at 90°C to prevent erosion ripples around the chip before developing. Then, the chip was soaked in the developer AZ300MIF and oscillated to remove the etched photoresist, and the PR was observed by a light aligner, a microscope, and a projection comparator, and then sent to do hard baking at 120°C for 15 min to fix the light, which prevents erosion during the moment. Soak the developed ITO in the etchant for approximately 10 min. The formula of etchant is hydrochloric acid 36%, nitric acid 70% and diH2O and the ratio is 50:3:50. The glass immersed in acetone and shook it to get rid of PR and the pattern of ITO electrode would be developed clearly. Finally, the glass substrate printed with the first electrode was placed in a 100% acetone shake to remove the photoresist, and the first electrode on the chip was completed.</p>
			<p> After confirming the state of the first electrode, the process of the second electrode will be performed on the same chip. The chip was immersed in acetone at the ultrasonic for 5 min to remove the organic particles or dust, then it was immersed in methanol at the ultrasonic for 5 min to remove acetone, then finally it was immersed in diH2O at ultrasonic for 5 min, which the chip was dried by nitrogen gun after completing the above clean process. Bake it for 15 min at 120°C to prevent the residual -OH from reacting with the PR. The side of electrode was applied HMDS to enhance PR adhesion before applied PR, which used spin coater in 3000 rpm 10 s at 1 step to attach the drug and 5000 rpm 20 s at 2 step to control the thickness of the drug. Note that the two drugs cannot produce any bubbles during the process of dropping the glass substrate. Sending PR-coated chip to do soft baking for 2 min at 90°C to let PR dry a little and enhance the adhesion. After the chip was soft baking, exposing the second mask, which have to align the cross pattern of the first pattern and irradiating at a 365 nm ultraviolet light with a mercury lamp for 2.5 s. The part is not covered by the mask and directly contacts the UV light will be eroded. Send the chip to do bake after exposure for 1 min at 90°C to prevent erosion ripples around the chip before developing. Then, the chip was soaked in the developer AZ300MIF and oscillated to remove the etched PR, and the PR was observed by a light aligner, a microscope, and a projection comparator to observe the erosion PR. A layer of approximately 1500 – 1600 Å of gold is then plated with a thermal evaporator. Before the gold plating, the machine must be vacuumed, and the surface of the chip electrode is plated with titanium under vacuum to strengthen the adhesion of gold. Then the solid gold is placed on the tungsten sheet in the evaporation machine, and the time of process is about 9-10 minutes. The gaseous gold will be solidification on the surface of the chip, and after the chip is taken out of the thermal evaporator, the acetone is removed to the PR, and the chip is completed.</p>
			<h3>Platform Electrode</h3>
			<p> First, titanium is plated on the silicon substrate by a thermal resistance vapor deposition machine to increase the affinity between Au and silicon substrate. Then evaporate a layer of gold from approximately 1500 – 1600 Å. The graphene growing on the PMMA membrane was then tiled on the gold, and the test chip was completed.</p>
			<h3>Cyclic Voltammetry</h3>
			<p> Before performing the individual connection test between the electrode material and the DNA, the chip must be cleaned with acetone and methanol separately. After the chip is packaged, the initial CV pattern of the chip can be obtained for the first measurement, and then dropped into the environment at 25 ° C. The ssDNA of 10 mM/1 mM/0.1 mM was rested for 16 hr for DNA ligation, and then the DNA was ligated to the chip to obtain a CV chart.</p>
			<p> The surface of the gold electrode needs to be modified to join the DNA. After the chip package is completed, it must first be cleaned by cyclic voltammetry.The first wash was carried out at 1 M NaOH at a speed of 0.3 V / s in the range -0.2 to + 0.8 V, followed by a second wash in 0.5 M H2SO4 at a speed of 0.3 V / s and a range of -0.25 to + 1.75 V.Next, SH modification of the surface of the gold electrode was carried out by placing 1,4-benzenedithiol 10 mM, 10 mL/each for 24 hours.</p>
			<h2>discussion</h2><hr>
			<h3>The time of exposion, development and eaching</h3>
			<h3>The distance between two electrodes</h3> 
			<p> It's an importamt part.</p> 
			<h3>The selection od material</h3> 
			<h3>The effect result from measurment</h3> 
			<p> The measurment may let DNA probe lift off.</p> 
			<h2>conclusion</h2><hr>
		</div>
		
		<div id="n3" class="w3-animate-opacity page" style="display:none">
			<h1>Detection</h1>
			<h2>introduction</h2><hr>
			<p> Cyclic Voltammetry</p>			
			<h2>experiment</h2><hr>				
			<h3>DNA-probe immobilization </h3>					
			<h3>DNA-target hybridization</h3>
			<h4>*target DNA concentration VS.probe DNA concentration</h4>
			<h4>*length of target DNA</h4>
			<h5>1000↑bp</h5>
			<h5>1000bp</h5>
			<h5>1000↓bp</h5>
			<h5>genomic DNA (positive VS. negative)</h5>					
			<h2>discussion</h2><hr>
			<h2>conclusion</h2><hr>
		</div>
	</div>
</div>
<script src="jq.js"></script><script>
	$(document).on('click', 'a[href^="#"]', function(event){event.preventDefault(); $('html, body').animate({scrollTop: $($.attr(this, 'href')).offset().top}, 700);});
	Acc=(x)=>{x = document.getElementById(x); x.className = x.className.indexOf("w3-show") == -1 ? x.className + " w3-show" : x.className.replace(" w3-show", "");}
	opL=(a)=>{x = document.getElementsByClassName("page"); for(i = 0; i < x.length; i++) x[i].style.display = "none"; color =" w3-pale-red";
		  x = document.getElementsByClassName("menu"); for(i = 0; i < x.length; i++) x[i].className = x[i].className.replace(color, "");
		      document.getElementById(a).style.display = "block"; event.currentTarget.className += color;}
</script>
