<title> ATCGU </title><meta charset="utf-8"><meta name="viewport" content="width=device-width, initial-scale=1"><link rel ="stylesheet" href="a.css"><link rel ="shortcut icon" href="logo.ico">
<style>img{display:inline}</style>
<div class="w3-top w3-show-inline-block w3-white w3-card" style="z-index:2">
	<div class="w3-hide-small w3-center">			
		<a style="width:99;color:#464646" class="w3-button w3-medium w3-padding-small"             href="Home.html"    >Home    </a>
		<a style="width:99;color:#464646" class="w3-button w3-medium w3-padding-small"             href="Project.html" >Project </a>
		<a style="width:99;color:#464646" class="w3-button w3-medium w3-padding-small"             href="Design.html"  >Design  </a>
		<a style="width:99;color:#464646" class="w3-button w3-medium w3-padding-small w3-pale-red" href="Lab.html"     >Lab     </a>
		<a style="width:99;color:#464646" class="w3-button w3-medium w3-padding-small"             href="Team.html"    >Team    </a>
		<a style="width:99;color:#464646" class="w3-button w3-medium w3-padding-small"             href="Sponsors.html">Sponsors</a>
	</div>
	<div class="w3-hide-large w3-hide-medium">
		<a   class="w3-bar-item w3-button w3-large w3-pale-red" href="Home.html">Home</a>
		<div class="w3-right w3-dropdown-hover">
			<a   class="w3-bar-item w3-button" href="#"><img src="logo.ico" style="width:100%;max-width:27"></a>
			<div class="w3-dropdown-content w3-bar-block w3-border" style="right:0">
				<a href="Project.html"  class="w3-bar-item w3-button w3-medium">Project </a>
				<a href="Design.html"   class="w3-bar-item w3-button w3-medium">Design  </a>
				<a href="Lab.html"      class="w3-bar-item w3-button w3-medium">Lab     </a>
				<a href="Team.html"     class="w3-bar-item w3-button w3-medium">Team    </a>
				<a href="Sponsors.html" class="w3-bar-item w3-button w3-medium">Sponsors</a>
			</div>
		</div>
	</div>
</div>
<div class="w3-animate-opacity" style="margin:0 5%">
	<div class="w3-col l2 m3 w3-card-4 w3-sidebar w3-bar-block w3-hide-small w3-animate-zoom w3-center" style="top:80;left:80;height:80%;padding-top:3%">
		<img class="w3-image w3-padding" src="logo.ico" style="width:80%">
		<a href="Lab.html"               class="w3-bar-item w3-button"          ><h1>Lab     </h1></a>
		<a href="#" onclick="Acc('dc')" class="w3-bar-item w3-button w3-large">Molecular Biology</a>
		<div id="dc" class="w3-hide w3-light-grey w3-show">
			<a href="#Introduction"  class="w3-button w3-small w3-half" style="color:gray">Introduction</a>
			<a href="#Experiment  "  class="w3-button w3-small w3-half" style="color:gray">Experiment  </a>
			<a href="#Discussion  "  class="w3-button w3-small w3-half" style="color:gray">Discussion  </a>
			<a href="#Conclusion  "  class="w3-button w3-small w3-half" style="color:gray">Conclusion  </a>
		</div>
		<a href="Semiconductor.html"     class="w3-bar-item w3-button w3-large">Semiconductor</a>
		<a href=    "Detection.html"     class="w3-bar-item w3-button w3-large">Detection    </a>
	</div>
	<div class="w3-col l9 m7 s12 w3-right">
		<div style="height:50"></div>
		<div class="w3-animate-opacity">
			<h1  class="w3-center">Molecular Biology</h1><hr>
			<div class="pt" id="Introduction"></div>
			<h2>Introduction</h2>
			<h3><li>
				<a style="cursor:pointer" onclick="w3m1.style.display='block'">MRSA</a>
			<span style="margin-left:3%">
				<a style="cursor:pointer" onclick="w3m1.style.display='block'"><img class="w3-mobile w3-hover-opacity" src="bridge.png" style="width:150"></a>
			</span>
			</li></h3>
			<div id="w3m1" class="w3-modal">
				<header class="w3-padding w3-modal-content w3-animate-top w3-card-4">
					<span onclick="w3m1.style.display='none'" class="w3-button w3-display-topright">×</span>
					<h1>Usa 300 epidemiology</h1><hr>
					<div class="content">
						<p>The community-associated MRSA strain USA300, which nearly always carries genes for the Panton-Valentine leukocidin (PVL) and the staphylococcal cassette chromosome mec (SCCmec) type IV, became the predominant strain type of MRSA circulating in the United States by 2011. Community-associated MRSA infections, defined as infections in patients who lacked recent exposure to the health care setting, disproportionately affected children , incarcerated populations, underserved urban populations, and other specific groups . Early US reports on community-associated MRSA infections were published from Houston , Chicago and elsewhere in the Midwest like Minnesota , Tennessee , Hawaii , and California. Soon after it began spreading in the community, USA300 became a common cause of infections in the health care setting as well, blurring the epidemiologic distinction between community-associated and health care–associated MRSA.</p>
						<p>Patients with USA300 MRSA bacteremia were more likely to have severe sepsis or septic shock in response to their infection than patients with non-USA300 MRSA bacteremia. The USA300 MRSA strain may be more virulent than other strains of MRSA in patients with MRSA infections complicated by bacteremia.</p>
						<p>The community-associated MRSA strain USA300 became the predominant strain type of MRSA circulating in the United States since 2011.</p>
						<p>In Taiwan, MRSA was first documented in the early 1980s . The incidence of hospital-associated MRSA infections increased remarkably in the 1990s . In the early 2000s, MRSA accounted for 53–83% of all S. aureus clinical isolates in most major hospitals, the predominant strain of CA-MRSA is sequence type ST59(SCCmec VT, PVL-positive) while HA-MRSA are ST239(SCCmec III,PVL-negative) and ST5(SCCmec II,PVL-negative) in 2000s. Since first report in 2002 , the rate of MRSA amongst childhood CA S. aureus infections increased significantly from 9.8% in 1999–2000  to 55.7% in 2004–2005. Since 2012, Emergence of MRSA USA300 has  been reported in several Asian countries . Recently, we identified four MRSA USA300  isolates from children with CA-MRSA infection in 2012 in our hospital,  prompting our study on this strain in Taiwan.</p>
						<p>Methicillin-resistant Staphylococcus aureus (MRSA) is a major pathogen in hospitals and the community. It causes a wide range of infections affecting the superficial skin and soft tissue, as well invasive infections including pneumonia, bloodstream infections, bone and joint infections. In the past, most of the infections caused by MRSA were confined to elderly patients in healthcare facilities and they were described as healthcare-associated MRSA (HA-MRSA). However, the epidemiology of MRSA changed overtime with the emergence of strains in patients with no previous history of hospitalization which were known as community-associated MRSA (CA-MRSA). CA-MRSA strains are more genetically diverse and belong to different clones than HA-MRSA. Some clonal complexes are epidemic and are spread internationally while others are restricted to certain regions. The spread of epidemic MRSA clones globally is of concern since it limits antibiotic options for treating MRSA infections.</p>
					</div>
				</header>
			</div>
			<p> Methicillin-resistant Staphylococcus aureus (MRSA) is a major pathogen in hospitals and the community. It causes a wide range of infections affecting the superficial skin and soft tissue, as well invasive infections including pneumonia, bloodstream infections, bone and joint infections.</p>
			<h3><li>Specificity (30-mer vs 999-mer)</li></h3>
			<p> With a single electrode, DNA biosensor can detect a DNA fragment of only 16~30-mer length because of the limitation by the length of probe. However, based on the properties that electrons can be transferred through the DNA, we use two electrodes with same probe length as single electrode, it can detect more than 1000-mer by analyzing the conductance. Besides, the target DNA needs to hybridize with two different probes to let electrons be transferred through the DNA.</p>			
			<div class="pt" id="Experiment"></div>
			<h2>Experiment</h2>
			<h3><li>Preparation of the Target DNA</li></h3>
			<ul>
			<h4><li>
				<a style="cursor:pointer" onclick="w3m2.style.display='block'">Genomic dna extraction</a>
			<span style="margin-left:3%">
				<a style="cursor:pointer" onclick="w3m2.style.display='block'"><img class="w3-mobile w3-hover-opacity" src="bridge.png" style="width:150"></a>
			</span>
			</li></h4>
			<div id="w3m2" class="w3-modal">
				<header class="w3-padding w3-modal-content w3-animate-top w3-card-4">
					<span onclick="w3m2.style.display='none'" class="w3-button w3-display-topright">×</span>
					<h1>Genomic dna extraction</h1><hr>
					<div class="content">
						<h3>Preprocess</h3>
						<ol>
							<li>Alter 1 ml USA300 culture to 1.5 ml microcentrifuge tube</li>
							<li>Centrifuge at 13000 rpm for 1 minutes at 25 ℃</li>
							<li>Add 100 ul TE buffer to the tube and pipetting the pellet</li>
							<li>Add  20 ul [100 mg/ml lysozyme and 10 mg/ml lysostaphin]</li>
							<li>Put in the 37 ℃ incubator for 60 minutes and gently shake it every 10 minutes</li>
						</ol>
						<h3>Cell lysis</h3>
						<ol>
							<li>Add 300 ul cell lysis buffer</li>
							<li>Put in the 60 ℃ incubator and gently shake it every 3 minutes, and cool down for 5 minutes</li>
						</ol>
							<h3>RNA removal</h3>
						<ol>
							<li>Add 1.5 ul RNase A and put at room temperature for 5 minutes</li>
						</ol>
							<h3>Protein removal</h3>
						<ol>
							<li>Add 100 ul protein remove buffer</li>
							<li>Put it on the ice for 10 minutes and precool the centrifuge</li>
							<li>Centrifuge at 13000 rpm for 10 minutes at 4 ℃</li>
						</ol>
							<h3>DNA precipitation</h3>
						<ol>
							<li>Place the supernatant to a new 1.5 ml microcentrifuge tube</li>
							<li>Add 300 ul isopropanol</li>
							<li>Centrifuge at 13000 rpm for 5 minutes at 4 ℃</li>
							<li>Add 300 ul 70% ethanol</li>
							<li>Centrifuge at 13000 for 3 minutes at 25 ℃</li>
							<li>Drain the ethanol at vacuum environment</li>
						</ol>
							<h3>DNA rehydration</h3>
						<ol>
							<li>Add 100 ul DNA hydration buffer</li>
							<li>Put in the 60 ℃ incubator for 60 minutes</li>
						</ol>
					</div>
				</header>
			</div>				
			<center><img src="7848.jpg" style="width:80%;height:45%"></center>
			<p>To ensure the quality of our DNA, we use nanodrop to make sure our genomic DNA is successfully extracted. The ratio of the absorbance at 260 and 280 nm (A260/280) is used to assess the purity of nucleic acids (Table. B). For pure DNA, A260/280 is widely considered at 1.8. so in our table, almost all sample we used are pure DNA. And A260/A230 also means the purity of nucleic acids but it is compare to like RNA or other substance. The best value should be 2, our DNA sample is located at 1.5-1.7. Although they aren’t perfect but are enough for our experiment.</p>
			<h4><li>
				<a style="cursor:pointer" onclick="w3m3.style.display='block'">Real-Time Polymerase Chain Reaction</a>
				<span style="margin-left:3%">
					<a style="cursor:pointer" onclick="w3m3.style.display='block'"><img class="w3-mobile w3-hover-opacity" src="bridge.png" style="width:150"></a>
				</span>
			</li></h4>
			<div id="w3m3" class="w3-modal">
				<header class="w3-padding w3-modal-content w3-animate-top w3-card-4">
					<span onclick="w3m3.style.display='none'" class="w3-button w3-display-topright">×</span>
					<h1>Real-Time Polymerase Chain Reaction</h1><hr>
					<div class="content">
						<ol>
							<li>deploy the solution containing 1 ul sample, 10 ul KAPA SYBR® FAST qPCR Master Mix (2X), 0.4 ul forward primer, 0.4ul reverse primer, 0.4 ul KAPA SYBR® FAST ROX Low Reference Dye (50X), and 7.8 ul DEPC water per tube.</li>
							<li>( USA300 + DEPC water ) * 3 repetition * 1 ( w553, w554 ) = 6 tubes.</li>
							<li>Run the qPCR machine.</li>
							<li>Denature at 95 ℃ for 3 minutes.</li>
							<li>40 cycles ( denature at 95 ℃ for 3 seconds, and annealing at 50 ℃ for 30 seconds ).</li>
                                                        <li>Denature at 95 ℃ for 15 seconds, annealing at 60 ℃ for 1 minute, and extension at 95 ℃ for 15 seconds.</li>
						</ol>
					</div>
				</header>
			</div>
        		<center><img src="ct.PNG"  style="width:95%"></center>
			<p>To ensure mecA is the specific gene in MRSA, we used 8 positive controls and 6 negative controls to run the real-time PCR machine with our primer. The CT means how many cycles does the DNA concentration reach in the threshold that the machine can detect. When the value is smaller, the DNA concentration or quantity is higher. Look at the table, there is two part in the table – positive ( left part ) and negative ( right part ). The CT of positive is between 14 to 23 cycles and negative is between 30 to 37 cycles. This mean the sequence we want in the positive is correctly magnify.</p>
			<p>Besides the CT in the table A, we also use melting curve to analyze our outcome by real-time PCR. Melting curve means the value that the temperature when half of the target DNA is annealed together. Look at the figure A, the curve position of positive control is different from negative control. And we use gel electrophoresis to prove the size of DNA is correct in positive control and incorrect in negative control. This experiment can prove our specific primer.</p>
			<h4><li>
				<a style="cursor:pointer" onclick="w3m4.style.display='block'">Polymerase Chain Reaction</a>
			<span style="margin-left:3%">
				<a style="cursor:pointer" onclick="w3m4.style.display='block'"><img class="w3-mobile w3-hover-opacity" src="bridge.png" style="width:150"></a>
			</span>
			</li></h4>
			<div id="w3m4" class="w3-modal">
				<header class="w3-padding w3-modal-content w3-animate-top w3-card-4">
					<span onclick="w3m4.style.display='none'" class="w3-button w3-display-topright">×</span>
					<h1>Polymerase Chain Reaction</h1><hr>
					<div class="content">
						<ol>
							<li>Deploy the solution which contain 2.5 ul sample, 2 ul forward primer, 2 ul reverse primer, 25 ul GoTaq®  Green Master Mix (2X), and 18.5 ul ddH2O per tube.</li>
							<li>Run the PCR machine.</li>
							<li>Denature at 94 ℃ for 3 minutes.</li>
                                                        <li>35 cycles ( denature at 94 ℃ for 30 seconds, annealing at 55 ℃ for 30 seconds and extension at 72 ℃ for 5 minutes ).</li>
							<li>Extension at 72 ℃ for 5 minutes</li>
						</ol>
					</div>			
				</header>
			</div>
			<p>PCR is the most important part of our molecular biology experiments, and it is very simple in principle, but it is indispensable. PCR is a tool to help verify the correctness of a sequence, whether it is for probe DNA or genomic DNA. In addition, it is conceptually related to our design philosophy. When PCR uses primer to test the sample sequence, it also symbolizes that our chips is double-checked from both ends in a more advanced way. Set up a bridge that belongs to DNA and set up a technology that connects the old and the new era.</p>
			<p>Number 1 to 8 are positive controls extracted from different strains of MRSA, the order is followed by SA40, SA957, LSA261, VA109, VB188, LTF01, USA300 and COL. Number 9 to 14 are negative controls and the order is followed by MSSA genomic DNA, Ecoli genomic DNA, Streptococcus pneumoniae genomic DNA, Rhizopus stolonifera genomic DNA, Penicillium genomic DNA, and water.</p>
			<p><center><img src="975target.PNG" class="w3-mobile" style=" width:70%"></center>
			<p> We found these DNA fragments, whether positive or negative control, by PCR. In the electropherogram, each band in positive control is observed to have the length of about 975bp, and there wasn't any band observed in negative control. DNA probes on chip FF is aimed to hybridize with the sequence, which is 975bp in length, contained in mecA gene. So, with this experiment, and the following gel extraction, we can proof the fact that the target DNA in length of 975bp can indeed hybridize with the DNA probes and then power on the current on chip FF. The result in negative control is to prove that mecA gene is specific in MRSA.</p>
			<center><img src="1277target.PNG"class="w3-mobile" style=" width:70%"></center>
			<p>We found these DNA fragments, whether positive or negative control, by PCR. In the electropherogram, each band in positive control is observed to have the length of about 1277bp, and there wasn't any band observed in negative control. DNA probes on chip FF is aimed to hybridize with the sequence, which is 975bp in length, contained in mecA gene. So, with this experiment, and the following gel extraction, we can test whether the target DNA in length of 1277bp can hybridize with the DNA probes and then power on the current on chip FF. The result in negative control is to prove that mecA gene is specific in MRSA.</p>
			<center><img src="175target.PNG" class="w3-mobile" style=" width:70%"></center>
			<p>This experiment used PCR to search for a 175 bp DNA fragment. It is Included in the 975 bp fragments detected by the probe in the mecA gene. The purpose is to help proving that even if the sample and the probe on the chips can smoothly hybrid, the length is not long enough, the chips still can not be conducted. After purifyinf and denaturing the DNA fragments, they can be used as a negative control for subsequent chips testing experiments.Numbers 9-14 are the negative control of this experiment, which indicates that the mecA gene is not present in their genomic DNA.</p>
			<center><img src="480target.PNG" class="w3-mobile" style=" width:70%"></center>
			<p> Since the probes at both ends of the chips is aimed at a fragment of 975 bp in the 2007 bp of the mecA gene, the design of this experiment is to find a mecA gene fragment other than the 975 bp fragment by PCR. We hope to get the fact that the samples have mecA gene can not make chips conductive after being purifying and denaturing. This will take as one of the negative controls of the subsequent chips test experiments. The total length of this gene fragment is 480 bp. In addition, numbers 9-14 are the negative comtrol of this experiment, which indicates that the mecA gene is not present in their genomic DNA.</p>
						<h4><li>
				<a style="cursor:pointer" onclick="w3m6.style.display='block'">DNA digestion</a>
				<span style="margin-left:3%">
					<a style="cursor:pointer" onclick="w3m6.style.display='block'"><img class="w3-mobile w3-hover-opacity" src="bridge.png" style="width:150"></a>
				</span>
			</li></h4>
			<div id="w3m6" class="w3-modal">
				<header class="w3-padding w3-modal-content w3-animate-top w3-card-4">
					<span onclick="w3m6.style.display='none'" class="w3-button w3-display-topright">×</span>
					<h1>DNA digestion</h1><hr>
					<div class="content">
						<h3>DNA digestion</h3>
						<ol>
							<li>culture the plasmid overnight.</li>
							<li>place 1 ml plasimid culture to a 1.5 ml microcentrifuge tube and centrifuge at 13000 for 1 mins.</li>
							<li>flow out the supernatant</li>
							<li>repeat step 2,3 for 3 times</li>
							<li>add 400 ul PD1 buffer and strongly pipet until the pellet resuspended.</li>
							<li>add 400 ul PD2 buffer and gently pipet it.</li>
							<li>add 600 ul PD3 buffer and gently pipet it.</li>
							<li>place the fiuld to a PG column and let the DNA bind for 5 mins.</li>
							<li>centrifuge at 13000 rpm for 5 mins.</li>
							<li>repeat step 8 for 3 times.</li>
							<li>flow out the supernatant.</li>
							<li>add 400 ul W1 buffer.</li>
							<li>centrifuge at 13000 rpm for 1 mins.</li>
							<li>flow out the supernatant.</li>
							<li>add 600 ul wash buffer.</li>
							<li>centrifuge at 13000 rpm for 1 mins.</li>
							<li>flow out the supernatant.</li>
							<li>centrifuge at 13000 rpm for 3 mins to dry out.</li>
							<li>put the PG column in a new microcentrifuge tube.</li>
							<li>add 50 ul ddH2O.</li>
							<li>centrifuge at 7000 rpm for 2 mins.</li>
							<li>repeat step 21 for 3 times</li>
							<li>add the 3.1 buffer, restriction enzyme we choose and ddH2O to digest the DNA overnight.>
							<li>do the gel electrophoresis to check the result</li>
							<li>gel extraction</li>
						</ol>
					</div>
				</header>
			</div>
			<p> To purified our target DNA not only by PCR, we also use DNA digestion to produce the DNA. We use PET-28 for our plasmid, and we choose two kinds of digestion. One is single digestion and the other is double digestion. And the restriction enzyme we use is EcoRV, XhoI and HincII. They can cut about 1400-1500 mers dor our DNA.</p>
				<h4><li>
				<a style="cursor:pointer" onclick="w3m5.style.display='block'">DNA Extraction</a>
				<span style="margin-left:3%">
					<a style="cursor:pointer" onclick="w3m5.style.display='block'"><img class="w3-mobile w3-hover-opacity" src="bridge.png" style="width:150"></a>
				</span>
			</li></h4>
			<div id="w3m5" class="w3-modal">
				<header class="w3-padding w3-modal-content w3-animate-top w3-card-4">
					<span onclick="w3m5.style.display='none'" class="w3-button w3-display-topright">×</span>
					<h1>DNA extraction</h1><hr>
					<div class="content">
						<h3>gel extraction</h3>
						<ol>
							<li>cut down the gel which contain the band we want</li>
							<li>add 500 ul buffer B and vortex</li>
							<li>put in the 60 ℃ incubator for 10 minutes and vortex it per 3 minutes</li>
							<li>cool down</li>
							<li>place the fluid to the PG column and centrifuge at 14000 rcf for 30 seconds, and remove the supernatant</li>
							<li>add 400 ul buffer W1 and centrifuge at 14000 rcf for 30 seconds, and remove the supernatant</li>
							<li>add 600 ul buffer W2 and centrifuge at 14000 rcf for 30 seconds, and remove the supernatant</li>
							<li>centrifuge at 14000 rcf for2 minutes</li>
							<li>put the PG column in a new microcentrifuge tube</li>
							<li>add 50 ul preheat ddH2O and put at the heating plate for 5 minutes at 55 ℃</li>
							<li>centrifuge at 14000 rcf for 3 minutes</li>
						</ol>
					</div>
				</header>
			</div>
			<p>We use gel extraction to isolate a desired fragment of intact DNA from an agarose gel following agarose gel electrophoresis.

				<h4><li>DNA Denaturation</li></h4>
				<p>When detection is processed on chip FF, the DNA in the sample will hybridize to the DNA probes on the electrode. Therefore, we must first denature the DNA in the sample from double-stranded to single-stranded, so that the complementary hybridization is available. The double-stranded DNA can be opened by heating the sample to 95 degrees for 10 minutes, and the sample contains ssDNA is immediately added into 2X SSC buffer and store in 4 degrees in order to keep it from renaturing.</p>
			<center><img style="width:75%;padding-top:100;max-width:800" src="55555.gif"></center>
			</ul>
			<h3><li>Preparation of Capture probe</h3></li>
			<p> In order to improve the convenience and stability of the device in the overall design, we have made some modifications at the two terminals of the DNA bridge, that were, the DNA probes. First, after measuring the electric current after power-on and analyzing the conductivity, we hope to observe the occurrence of conduction more intuitively. Therefore, the 3' and 5' terminals of the probes are respectively connected with the fluorescent substances Cy3 and Cy5 to confirm the correctness of experiment result again. In addition, in order to make a more stable binding of the DNA probes to the electrodes, the gold electrode is modified with SH, and the graphene is modified with NH2 to enhance the binding between the probes and the electrodes.</p>
			<p class="w3-center">capture probe 1: 5'  HSC6-AAAATCGATGGTAAAGGTTGGCAAAAAGATAAA-Cy3  3'</p><img src="871.PNG">
			<p class="w3-center">capture probe 2: 5'  Cy5-AACTACGGTAACATTGATCGCAACGTTCAATTT-AminoC7  3'</p><img src="872.PNG">

			<div class="pt" id="Discussion"></div>
			<h2>Discussion</h2>
			<p>In the experimental content, we determined the presence of mecA gene in various strains of MRSA by PCR results to reflect the credibility of the subsequent detection results and confirm the corresponding probe sequences. There are many reasons for choosing PCR. First, PCR is currently the most widely used method for gene detection. Almost all related literatures for detecting MRSA genes use PCR. In addition, the most important reason is that the chips we designed are designed to surpass existing technology. Therefore, PCR is naturally the technology we expect to surpass. PCR could be chosen for the best comparison. The results of qPCR corroborate the specificity of our designed primers for mecA gene, and the purpose is also that support the correctness of the test results. In the part of molecular biology experiments, all experimental designs and results use to demonstrate the validity of the materials we use on the chips. At present, mecA is the only target gene, and in the future, these technologies can be used to continue to develop more probe sequences capable of solving the drug resistance genes harmful to humans, and achieve the final detection purpose.</P>
			
			<div class="pt" id="Conclusion"></div>
			<h2>Conclusion</h2>
			<p>Different lengths of target DNA from MRSA are prepared by using polymerase chain reaction and DNA extraction. Furthermore, we used polymerase chain reaction to confirm that our DNA probe is specific to our target DNA. Therefore, we can assume that our DNA probe will work on the “FF”.</P>
			</div>
		</div>		
	</div>
</div>
<div class="w3-center w3-col" style="color:#dac2c2;background-color:#464646"> © Copyright 2018 @CGU </div>
<script src="jq.js"></script><script>
	$(document).on('click', 'a[href^="#"]', function(event){event.preventDefault(); $('html, body').animate({scrollTop: $($.attr(this, 'href')).offset().top}, 700);});
	Acc=(x)=>{x = document.getElementById(x); x.className = x.className.indexOf("w3-show") == -1 ? x.className + " w3-show" : x.className.replace(" w3-show", "");}
	window.onclick = () => {
		if (event.target == w3m1) w3m1.style.display = "none";
		if (event.target == w3m2) w3m2.style.display = "none";
		if (event.target == w3m3) w3m3.style.display = "none";
		if (event.target == w3m4) w3m4.style.display = "none";
		if (event.target == w3m5) w3m5.style.display = "none";
		if (event.target == w3m6) w3m6.style.display = "none";
	}
</script>
